libs/documentation/src/lib/components/collapse/demos/basic/collapse-basic.component.scss

```diff
@@ -0,0 +1,7 @@
+.storybook-collapse--active {
```

Comment:
There should be classes associated with the existing functionality, those should used rather than defining custom ones for the demo. With this, both demo and component classes would have to keep in sync,  if a bug that needs to be fixed comes up two places rather than one need to be updated.